tests/ottest/fixtures/blockchain/RegtestSimple.cpp

```diff
@@ -640,8 +641,7 @@ void Regtest_fixture_simple::advance_blockchain(
     std::vector<std::unique_ptr<ScanListener>> scan_listeners;
     std::vector<std::unique_ptr<std::future<void>>> external_scan_listeners;
     std::vector<std::unique_ptr<std::future<void>>> internal_scan_listeners;
-    // Safety mechanism
-    ot::Sleep(100ms);
```

Comment:
we do not need this sleep here anymore ?

Reply:
No it is need because bug in ScanListener with wrong initialization callback and map. Now it should work without this sleep